### PR TITLE
[WIP] The Asciidoc examples fixed by hand

### PR DESCRIPTION
**Pull request by abhi18av**
Hi @awkay, 

This PR isn't really meant to be merged but just to have a backup of the example strings in the `adoc` format.  This is the hand-fixed (via vim macros) version of all the examples which I've completed as of now, there are still a few which are remaining and I'll cover them up as well.

If possible, please just give a glimpse to this to correct the `anchor tag` names for the examples. Your reference tags might be helpful in the adoc extension as well.

Regarding the automated `Asciidoc` extension, I'm tracking that here one https://github.com/abhi18av/FulcroExamplesAsciidoc/issues/1  

I'll go through it soon and when it's and will push a PR to add the extension to this repo as well. Let me know if you have any suggestions 😊

### CODE REVIEW

File: handmade-example-strings.adoc [hunk 1]
```diff
@@ -0,0 +1,681 @@
...
+----
+
+====
+
+
+;;;;;;;;;;;;;;;;;;;
+
+
+
+.[[]]<<,Input Focus and React Refs/Lifecycle>>
```
Comment (awkay) [inline handmade-example-strings.adoc]: ???

File: handmade-example-strings.adoc [hunk 2]
```diff
@@ -0,0 +1,681 @@
...
+----
+include::src/book/book/ui/focus_example.cljs[]
+----
+
+====
+
+;;;;;;;;;;;;;;;;;
+
+.[[]]<<,Drawing in a Canvas>>
+====
```
Comment (awkay) [inline handmade-example-strings.adoc]: A lot of these look corrupt to me

File: handmade-example-strings.adoc [hunk 3]
```diff
@@ -0,0 +1,681 @@
...
+[source,clojure,role="source"]
+----
+include::src/book/book/queries/dynamic_queries.cljs[]
+----
+
+====
+
+;;;;;;;;;;;;;;;;;
+
+.[[]]<<,Dynamic Query Parameters>>
```
Comment (awkay) [inline handmade-example-strings.adoc]: Yeah, these all need generated anchors and links

File: handmade-example-strings.adoc (hunk 1 above)
Comment (abhi18av) [reply handmade-example-strings.adoc]: I think this meant the absence of the anchors 😅

File: handmade-example-strings.adoc (hunk 2 above)
Comment (abhi18av) [reply handmade-example-strings.adoc]: Yup, these might already be resolved in the latest push submitted here

File: handmade-example-strings.adoc (hunk 3 above)
Comment (abhi18av) [reply handmade-example-strings.adoc]: Yes, I've fixed most parts but I do need some pointers regarding the `PreMerge-*` examples towards the end of the documents regarding the naming of the anchors. If possible please do give a glance at those as well.

**Comment by abhi18av** [comment]
With this PR my hope is that, even though the solution isn't ideal as of now, we can still move forward.

Once the changes here are confirmed, I'll start with the https://github.com/abhi18av/FulcroExamplesAsciidoc at once.

File: handmade-example-strings.adoc (hunk 2 above)
Comment (awkay) [reply handmade-example-strings.adoc]: Great, yeah, that looks better :)

**Comment by awkay** [comment]
Hey there, yes, these examples look like what I was hoping for...are you going to integrate them into the book next?  won't be working on the book for the next 48 hours probably, FYI, so if you got time now would be ideal.

**Comment by abhi18av** [comment]
Hi @awkay , that's good to know! Yes, today I plan to work on fulcro book.

Here's what I have in mind, 
- Go through the updated version of the book
- Replace the older examples with these links
- Push the PR

**Comment by awkay** [comment]
Looking good. If that's complete, then you should remove your extra file from the PR :)

**Comment by abhi18av** [comment]
@awkay , I am done with the extension now 😊 
https://github.com/abhi18av/FulcroExamplesAsciidoc

Initially I started with the manual transformation but it still is cognitively heavy so decided to invest some more time in the directions you had pointed earlier.

**Comment by abhi18av** [comment]
Here's how it look 

Screen Shot 2019-07-01 at 00 31 55

Screen Shot 2019-07-01 at 00 32 05

**Comment by abhi18av** [comment]
Next Steps:
- Go through the `DeveloperGuide.adoc` and replace all examples
- Update the `Makefile`

**Comment by awkay** [comment]
Nice! Looks good. Thanks for looking back at that...the macro solution is very much nicer, and ends up ok in PDF, right, because it is just preprocessor...

**Comment by abhi18av** [comment]
Yup, that's true!

I've made a PR here https://github.com/fulcrologic/fulcro-developer-guide/pull/13 and I'm closing this handmade examples PR in favor of the `Asciidoctor extension` based PR.

**Pull request closed without merging**